src/components/ErrorMassage/ErrorMassage.tsx

```diff
@@ -0,0 +1,24 @@
+type Props = {
+  errorMassage: string;
+  hideError: () => void;
+};
+
+export const ErrorMassage: React.FC<Props> = ({ errorMassage, hideError }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={`notification is-danger is-light has-text-weight-normal ${errorMassage.length > 0 ? '' : 'hidden'}`}
```

Comment:
You need to use the classnames library for add classes with condition, fix it everywhere